packages/shorebird_code_push_protocol/lib/src/messages/get_release_patches/get_release_patches_response.dart

```diff
@@ -0,0 +1,23 @@
+import 'package:json_annotation/json_annotation.dart';
+import 'package:shorebird_code_push_protocol/shorebird_code_push_protocol.dart';
+
+part 'get_release_patches_response.g.dart';
+
+/// {@template get_release_patches_response}
+/// The response to /api/v1/apps/$appId/releases/$releaseId/patches
+/// {@endtemplate}
+@JsonSerializable()
+class GetReleasePatchesResponse {
+  /// {@macro get_release_patches_response}
+  const GetReleasePatchesResponse({required this.patches});
+
+  /// Converts a Map<String, dynamic> to a [GetReleasePatchesResponse]
+  factory GetReleasePatchesResponse.fromJson(Map<String, dynamic> json) =>
+      _$GetReleasePatchesResponseFromJson(json);
+
+  /// Converts a [GetReleasePatchesResponse] to a Map<String, dynamic>
+  Json toJson() => _$GetReleasePatchesResponseToJson(this);
+
+  /// Patch numbers for a given release mapped to their artifacts.
+  final Map<int, List<PatchArtifact>> patches;
```

Comment:
Does this need to be a `Map`? Couldn't this be a `List<PatchArtifact>` since the patchId is part of the artifact?

Reply:
The artifact doesn't know which patch number (not ID) it's related to, and I think we probably want to display patch numbers on the client.

Reply:
Ah yeah nvm I misread makes sense 👍